firstvoices/backend/models/managers.py

```diff
@@ -1,17 +0,0 @@
-from django.db import models
```

Comment:
I moved this to the permissions folder, mainly so I could add a README in there that explains how the different parts work together